### PR TITLE
Fix migration

### PR DESCRIPTION
**Pull request by nickpell**
Fixes an issue where trial software plans are not being initialized.  Affects newly created instances of HQ.

Reuses the correct management command instead of copy/pasting code, and `django.apps` to get the correct models.  Best read commit by commit.

@orangejenny @biyeun

### CODE REVIEW

**Comment by nickpell** [comment]
If we really want to be careful about this the correct thing to do is to create a new migration ... I'm not too worried because the prod machines already have the correct models and fixing it on a dev's machine is as simple as running the right management command

**Comment by biyeun** [comment]
Looks good.. @nickpell did this issue only affect local dev environments, or could any of the newer servers (maybe zambia?) have been affected?

**Comment by nickpell** [comment]
only reported on @calellowitz 's machine.  If zambia didn't already have the plans set up, I think we would have gotten the "did you bootstrap plans?" error by now

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -589,7 +589,7 @@ def get_default_plan_by_domain(cls, domain, edition=None, is_trial=False):
             )
             return default_product_plan.plan.get_version()
         except DefaultProductPlan.DoesNotExist:
-            raise AccountingError("No default product plan was set up, did you forget to bootstrap plans?")
+            raise AccountingError("No default product plan was set up, did you forget to run cchq_software_plan_bootstrap?")
```
Comment (dimagimon) [inline corehq/apps/accounting/models.py]: E501 line too long (124 > 115 characters)

**Comment by orangejenny** [comment]
looks good to me

**Comment by biyeun** [comment]
:+1:

**Comment by biyeun** [comment]
(waiting for build)

**Comment by nickpell** [comment]
:white_check_mark:

**Pull request merged**